src/site/pages/modifiers.md

```diff
@@ -1,35 +1,61 @@
+- pf-m-2xl
+- pf-m-3xl
+- pf-m-4xl
+- pf-m-action
 - pf-m-active
 - pf-m-align-right
 - pf-m-all
+- pf-m-ascending
 - pf-m-block
-- pf-m-border
 - pf-m-bottom
+- pf-m-collapsed
+- pf-m-compact
+- pf-m-current
 - pf-m-danger
 - pf-m-dark
+- pf-m-dark-100
+- pf-m-dark-200
+- pf-m-descending
 - pf-m-disabled
+- pf-m-end
 - pf-m-error
 - pf-m-expanded
+- pf-m-fit-content
+- pf-m-flex-2
+- pf-m-flex-3
+- pf-m-flex-4
+- pf-m-flex-5
 - pf-m-focus
-- pf-m-form-control-alt
```

Comment:
Do we want to keep this if we are keeping the alt form control styles?

Reply:
nope!